clients/venice-push-job/build.gradle

```diff
@@ -55,6 +55,11 @@ dependencies {
     exclude group: 'org.apache.hadoop', module: 'hadoop-client-api'
   }
 
+  // Spark versions 3.2.X - 3.3.X are compiled with antlr4 4.8. In our classpath, antlr4 version 4.5 is used. This
+  // discrepancy causes errors at runtime.
```

Comment:
Hmm, that is weird... if we don't have an explicit dependency on antlr, then why isn't Gradle resolving the transitive deps to the highest one requested automatically? Anyway, it's not a blocker to merge, but I'm just curious...

Reply:
Yeah, it was surprising to me too. I saw that the version for `antlr4-runtime` was updated ot 4.8 transitively. However, `pegasus` (r2/d2) also pulls in `antlr4` module (version 4.5). We can also explore excluding this specific package if we find that it has no business at runtime (like `errorprone`/`checkerframework`/etc)

Reply:
I'm all for dep exclusion... we carry way too much baggage. Pegasus in particular is pretty heavyweight. Oh well.